src/components/Dashboard/DashboardGrid.jsx

```diff
@@ -2,11 +2,11 @@ import React, { useMemo, useCallback } from 'react';
 import PropTypes from 'prop-types';
 import { Responsive, WidthProvider } from 'react-grid-layout';
 import styled from 'styled-components';
+import some from 'lodash/some';
```

Comment:
This is native in Javascript now: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
in this case I want the simpler predicate to make the code a little more readable 
![image](https://user-images.githubusercontent.com/6663002/73476479-eca9fb00-4357-11ea-89e6-bb86790fedcb.png)